packages/experimental-utils/src/ts-eslint-scope/index.ts

```diff
@@ -9,4 +9,5 @@ export * from './Referencer';
 export * from './Scope';
 export * from './ScopeManager';
 export * from './Variable';
+// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
 export const version: string = ESLintVersion;
```

Comment:
weird - this shouldn't be `any` - we have an explicit type for it

https://github.com/typescript-eslint/typescript-eslint/blob/20f05f659d5cb219cfc7f7212679c0a17811b3e7/packages/experimental-utils/typings/eslint-scope.d.ts#L59-L63